src/main/java/net/dv8tion/jda/internal/requests/restaction/ForumPostActionImpl.java

```diff
@@ -132,6 +135,16 @@ public ForumPostAction setAutoArchiveDuration(@Nonnull ThreadChannel.AutoArchive
         this.autoArchiveDuration = autoArchiveDuration;
         return this;
     }
+    
+    @Nonnull
+    @Override
+    public ForumPostAction setSlowmode(int slowmode)
```

Comment:
After some testing, I noticed that this requires the `MANAGE_THREADS` permission, so we need to add a permission check here (and in the other impl as well):

```java
if (!getGuild().getSelfMember().hasPermission(channel, Permission.MANAGE_THREADS))
    throw new InsufficientPermissionException(channel, Permission.MANAGE_THREADS);
```

Documented like so

```java
* @throws net.dv8tion.jda.api.exceptions.InsufficientPermissionException
*         If the bot does not have {@link Permission#MANAGE_THREADS} in the parent channel
```

Reply:
I see, added it now, thanks.